object/datastore.go

```diff
@@ -58,6 +58,8 @@ func (e DatastoreNoSuchFileError) Error() string {
 
 type Datastore struct {
 	Common
+
+	DatacenterPath string
```

Comment:
Given everything lives in the context of a datacenter, should this move into Common eventually?